terra-app-setup-chart/templates/network-policy.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: deny-from-other-namespaces
+  namespace: {{ .Release.Namespace  }}
```

Comment:
We get this for free without any changes at install time leo-side, see https://helm.sh/docs/chart_template_guide/builtin_objects/